nextjs/src/components/ui/form.tsx

```diff
@@ -142,17 +142,29 @@ const FormDescription = React.forwardRef<
 FormDescription.displayName = "FormDescription";
 
 const FormMessage = React.forwardRef<
-  HTMLParagraphElement,
-  React.HTMLAttributes<HTMLParagraphElement>
->(({ className, children, ...props }, ref) => {
-  const { error, formMessageId } = useFormField();
-  const body = error ? String(error?.message) : children;
+ HTMLParagraphElement,
+ React.HTMLAttributes<HTMLParagraphElement> & { htmlContent?: string }
+>(({ className, children, htmlContent, ...props }, ref) => {
+ const { error, formMessageId } = useFormField();
+ const body = error ? String(error?.message) : children;
 
-  if (!body) {
+ if (!body && !htmlContent) {
     return null;
-  }
-
-  return (
+ }
+
+ if (htmlContent) {
```

Comment:
question: did you manually edit this file? I think it's a generated component, so it might be overwritten in the future. If you regenerated it with some other option, all good.

suggestion: revert this file, and instead find another approach that doesn't require the `htmlContent` prop. In fact, I think you can do this:

```javascript
<FormMessage>
    <span>Password reset successfully! You can now</span>
    <a href="/login">login</a>
    <span>.</span>
</FormMessage>
```

Reply:
oh nice - that's done